src/AutoRest.CSharp/DataPlane/Output/DataPlaneRestClient.cs

```diff
@@ -56,46 +43,44 @@ public DataPlaneRestClient(OperationGroup operationGroup, BuildContext<DataPlane
 
         protected override Dictionary<ServiceRequest, RestClientMethod> EnsureGetNextPageMethods()
```

Comment:
How different are the override and the base implemenataion?

Reply:
Diffing suggests no difference. Collapsing.